migrator/migrations/m_109_to_m_110_replace_deprecated_resources/migration_test.go

```diff
@@ -0,0 +1,541 @@
+				Integration:         storage.Access_READ_ACCESS,
+				// To-be-replaced-later resources
+				AllComments:           storage.Access_READ_ACCESS,
+				ComplianceRuns:        storage.Access_READ_ACCESS,
+				ComplianceRunSchedule: storage.Access_READ_ACCESS,
+				Config:                storage.Access_READ_ACCESS,
+				DebugLogs:             storage.Access_NO_ACCESS,
+				NetworkGraphConfig:    storage.Access_READ_ACCESS,
+				ProbeUpload:           storage.Access_READ_ACCESS,
+				ScannerBundle:         storage.Access_READ_ACCESS,
+				ScannerDefinitions:    storage.Access_READ_ACCESS,
+				SensorUpgradeConfig:   storage.Access_READ_ACCESS,
+				ServiceIdentity:       storage.Access_READ_ACCESS,
+				// Non-replaced resources
+				Alert:                            storage.Access_READ_ACCESS,
+				CVE:                              storage.Access_READ_ACCESS,
+				Cluster:                          storage.Access_READ_ACCESS,
+				Deployment:                       storage.Access_READ_ACCESS,
+				Detection:                        storage.Access_READ_ACCESS,
+				K8sRole:                          storage.Access_READ_ACCESS,
+				K8sRoleBinding:                   storage.Access_READ_ACCESS,
+				K8sSubject:                       storage.Access_READ_ACCESS,
+				Namespace:                        storage.Access_READ_ACCESS,
+				NetworkGraph:                     storage.Access_READ_ACCESS,
+				NetworkPolicy:                    storage.Access_READ_ACCESS,
+				Node:                             storage.Access_READ_ACCESS,
+				Policy:                           storage.Access_READ_ACCESS,
+				Role:                             storage.Access_READ_ACCESS,
+				Secret:                           storage.Access_READ_ACCESS,
+				ServiceAccount:                   storage.Access_READ_ACCESS,
+				VulnerabilityManagementApprovals: storage.Access_READ_ACCESS,
+				VulnerabilityManagementRequests:  storage.Access_READ_ACCESS,
+				VulnerabilityReports:             storage.Access_READ_ACCESS,
+				WatchedImage:                     storage.Access_READ_ACCESS,
+				// Internal resources
+				ComplianceOperator: storage.Access_READ_ACCESS,
+				InstallationInfo:   storage.Access_READ_ACCESS,
+				Version:            storage.Access_READ_ACCESS,
+			},
+		},
+	}
+)
+
+type psMigrationTestSuite struct {
+	suite.Suite
+
+	db *rocksdb.RocksDB
+}
+
+func TestMigration(t *testing.T) {
+	suite.Run(t, new(psMigrationTestSuite))
+}
+
+func (suite *psMigrationTestSuite) SetupTest() {
+	suite.db = rocksdbtest.RocksDBForT(suite.T())
+}
+
+func (suite *psMigrationTestSuite) TearDownTest() {
+	rocksdbtest.TearDownRocksDB(suite.db)
+}
+
+func (suite *psMigrationTestSuite) TestMigration() {
+
+	for _, initial := range UnmigratedPermissionSets {
+		data, err := proto.Marshal(initial)
+		suite.NoError(err)
+
+		key := rocksdbmigration.GetPrefixedKey(prefix, []byte(initial.GetId()))
+		suite.NoError(suite.db.Put(writeOpts, key, data))
```

Comment:
nit: we might want to stop doing anything afterwards if an error occurred here.
```suggestion
		suite.Require().NoError(suite.db.Put(writeOpts, key, data))
```